dts/bindings/input/kbd-matrix-common.yaml

```diff
@@ -23,6 +23,12 @@ properties:
       Defines the poll period in msecs between between matrix scans, set to 0
       to never exit poll mode. Defaults to 5ms if unspecified.
 
+  stable-poll-period-ms:
+    type: int
+    description: |
+      Defines the poll period in msecs between between matrix scans when the
```

Comment:
Please, remove the repeated "between" word from the sentence

Reply:
hey thanks, still working on this btw, need to do more testing and update the docs, no rush :-)

Reply:
fyi ended up only keeping the first patch, unconvinced about the second one, may pick it up again down the road